src/utils/querybuilder.util.js

```diff
@@ -514,16 +515,49 @@ const quantityQueryBuilder = function ({ target, field }) {
                 qB[`${field}.value`] = { $gte: num };
                 break;
             case 'ne':
-                qB[`${field}.value`] = { $ne: num };
+                offset = calcRange(num);
+                num = Number(num);
+                qB[`${field}.value`] = { $ne: { $gt: num - offset, $lt: num + offset }};
```

Comment:
Instead of 'gt' it will be 'gte' at both places because as per FHIR specification for '100' input, the range will be {'$gte':'99.5','$lt':'100.5'}

Reply:
Fixed